lib/nimble_template/projects/project.ex

```diff
@@ -80,6 +80,14 @@ defmodule NimbleTemplate.Projects.Project do
     |> List.first()
   end
 
+  @spec prepend_optional_addon(__MODULE__.t(), atom()) :: __MODULE__.t()
+  def prepend_optional_addon(
+        %__MODULE__{optional_addons: existing_addons} = project,
+        addon_module
+      ) do
+    Map.put(project, :optional_addons, [addon_module | existing_addons])
+  end
```

Comment:
```suggestion
      ), do: Map.put(project, :optional_addons, [addon_module | existing_addons])
```

Reply:
Updated in [9782e2e](https://github.com/nimblehq/elixir-templates/pull/292/commits/9782e2ef5fef654f7a75a5bf77f6729d46406c43)